impeller/entity/contents/content_context.cc

```diff
@@ -162,16 +162,19 @@ static std::unique_ptr<PipelineT> CreateDefaultPipeline(
 
 ContentContext::ContentContext(
     std::shared_ptr<Context> context,
-    std::shared_ptr<TypographerContext> typographer_context)
+    std::shared_ptr<TypographerContext> typographer_context,
+    std::shared_ptr<RenderTargetAllocator> render_target_allocator)
```

Comment:
This change allows tests to provide their own RenderTargetAllocator implementation, which I use to introspect into texture allocation.